tests/test_other.py

```diff
@@ -7919,12 +7919,12 @@ def test(filename, expectations, size_slack):
 
       print('test on hello world')
       test(path_from_root('tests', 'hello_world.cpp'), [
-        ([],      23, ['assert'], ['waka'], 46505,  24,   16, 59), # noqa
-        (['-O1'], 18, ['assert'], ['waka'], 12630,  16,   14, 31), # noqa
-        (['-O2'], 18, ['assert'], ['waka'], 12616,  16,   14, 31), # noqa
-        (['-O3'],  7, [],         [],        2690,  10,    2, 21), # noqa; in -O3, -Os and -Oz we metadce
-        (['-Os'],  7, [],         [],        2690,  10,    2, 21), # noqa
-        (['-Oz'],  7, [],         [],        2690,  10,    2, 21), # noqa
+        ([],      24, ['assert'], ['waka'], 46505,  26,   17, 62), # noqa
+        (['-O1'], 18, ['assert'], ['waka'], 12630,  17,   15, 34), # noqa
+        (['-O2'], 18, ['assert'], ['waka'], 12616,  17,   15, 34), # noqa
+        (['-O3'],  7, [],         [],        2706,  10,    2, 22), # noqa; in -O3, -Os and -Oz we metadce
+        (['-Os'],  7, [],         [],        2706,  10,    2, 22), # noqa
+        (['-Oz'],  7, [],         [],        2706,  10,    2, 22), # noqa
```

Comment:
looks like some extra function survives metadce, so it's actually used - what is it?

Reply:
It's `nullFunc_jiji`. We have one more unique signature now.

Reply:
I'm confused, I don't see any nullFunc methods when I build say `./emcc tests/hello_world.c -Os -profiling`. Reading emscripten.py those should only be emitted when ASSERTIONS are on. So that doesn't explain the change in the metadce levels here, `-O3, -Os, -Oz`?